src/content/operations/usage/_index.en.md

```diff
@@ -0,0 +1,854 @@
++++
+title = "User Guide"
+date = 2020-05-04T19:01:14+05:30
+weight = 15
++++
+
+## Overview
+
+This guide is intended for users who have a Submariner environment set up and want to verify the installation and learn more about how to
+use Submariner and the main capabilities it provides. This guide assumes that there are two Kubernetes clusters,
+**cluster2** and **cluster3**, forming a clusterset, and that the Broker is deployed into a separate cluster **cluster1**.
+
+{{% notice tip %}}
+Make sure you have `subctl` [set up](../deployment/subctl/#installation). Regardless of how Submariner was deployed, `subctl` can be used
+for various tasks such as verification and troubleshooting, as shown in this guide.
+{{% /notice %}}
+
+{{% notice note %}}
+This guide focuses on a non-Globalnet Submariner deployment.
+{{% /notice %}}
+
+### 1. Validate the Installation
+
+#### On the Broker
+
+The Broker facilitates the exchange of metadata information between the connected clusters, enabling them to discover one another.
+The Broker consists of a set of Custom Resource Definitions (CRDs) only; there are no Pods or Services deployed with it.
+
+You can use this command to validate that the Broker namespace has been created in the Broker cluster:
+
+```bash
+$ kubectl get namespace submariner-k8s-broker
+NAME                    STATUS   AGE
+submariner-k8s-broker   Active   5m
+```
+
+You can use this command to validate that the Submariner CRDs have been created in the Broker cluster:
+
+```bash
+$ kubectl get crd | grep submariner
+clusters.submariner.io                    2020-11-30T13:49:16Z
+endpoints.submariner.io                   2020-11-30T13:49:16Z
+gateways.submariner.io                    2020-11-30T13:49:16Z
+serviceimports.lighthouse.submariner.io   2020-11-30T13:49:16Z
+```
+
+You can use this command to validate that the participating clusters have successfully joined the Broker:
+
+```bash
+$ kubectl -n submariner-k8s-broker get clusters.submariner.io
+NAME       AGE
+cluster2   5m9s
+cluster3   2m9s
+```
+
+#### On Connected Clusters
+
+The below commands can be used on either **cluster2** or **cluster3** to verify that the two clusters have successfully formed a clusterset
```

Comment:
Should also add step to verify CRD installation on connected clusters. Just because they're all installed correctly on broker doesn't mean they might be on connected clusters too. Also, on connected clusters we need to make sure `serviceexports` CRD is installed even if it is not needed on broker.